internal/app/cf-terraforming/cmd/generate.go

```diff
@@ -275,6 +275,30 @@ func generateResources() func(cmd *cobra.Command, args []string) {
 				}
 				jsonStructData[i].(map[string]interface{})["secret"] = secret
 			}
+		case "cloudflare_user_agent_blocking_rule":
+			page := 1
+			var jsonPayload []cloudflare.UserAgentRule
+			for {
+				res, err := api.ListUserAgentRules(context.Background(), zoneID, page)
+				if err != nil {
+					log.Fatal(err)
+				}
+
+				jsonPayload = append(jsonPayload, res.Result...)
```

Comment:
just FYI, this isn't concurrency safe. not sure if that is factor for you, but worth noting.

Reply:
Thanks for that! I pulled this idea from pagination examples in cloudflare-go. I think we should be OK with this until we don't introduce concurrency here.